nautobot/core/testing/views.py

```diff
@@ -737,6 +739,32 @@ def get_title(self):
         def get_list_view(self):
             return lookup.get_view_for_model(self.model, view_type="List")
 
+        def test_table_with_indentation_is_removed_on_filter_or_sort(self):
```

Comment:
As @whitej6 brought up in standup, let's make sure we document this new generic test case in the changelog and send a heads-up to app developers.

Reply:
What do you think the category should be? Added, changed or housekeeping? Probably added or changed for the best visibility.

Reply:
I'd go with `added`